### ISSUE TITLE
MAISI Inference

### ISSUE THREAD

**Issue by Zeyad-Zaky** [Project-MONAI/tutorials#1940]
I am trying to use MAISI inference and I have a couple of questions.

my pipeline now to generate a segmentation mask to give as input for MAISI inference is:
1. generate segmentation mask using TotalSegmentatorV2
2. generate segmentation mask using Visat3D
3. unify labels (convert TS labels to same values of Vista3D)
4. generate unified mask to take best segmentation
5. use MAISI inference

my questions are:
1. is that pipeline helps to get best results or it has some problems, as I could not neither find in the paper nor documentations how you prepare the segmentation masks.
2. Important question: both segmentation networks do not provide segmentation masks for air or body regions. I investigated the training dataset and found that the value 200 is assigned to those parts, however I could not find anywhere what is HU threshold you are using, so please also if you can guide onto that part whether where to find it or what is the threshold being used.

**Comment by Can-Zhao** [comment]
@guopengf May I ask if you would be able to explain the details? Thanks!
